app/js/components/manageApps/ManageApps.jsx

```diff
@@ -415,6 +417,30 @@ export default class ManageApps extends React.Component {
     };
   }
 
+  checkForUpdates() {
+    const installedAddons = this.state.appList;
+    const updatesAvailable = [];
+    axios.get(`https://addons.openmrs.org/api/v1//addon`).then((response) => {
```

Comment:
`//addon` instead of `/addon` in the url.

Reply:
Will change that right away.Thanks